src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.ShoppingCartResponseMapper;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final UserService userService;
+    private final MovieSessionService movieSessionService;
+    private final ShoppingCartService shoppingCartService;
+    private final ShoppingCartResponseMapper shoppingCartResponseMapper;
+
+    public ShoppingCartController(UserService userService,
+                                  MovieSessionService movieSessionService,
+                                  ShoppingCartService shoppingCartService,
+                                  ShoppingCartResponseMapper shoppingCartResponseMapper) {
+        this.userService = userService;
+        this.movieSessionService = movieSessionService;
+        this.shoppingCartService = shoppingCartService;
+        this.shoppingCartResponseMapper = shoppingCartResponseMapper;
+    }
+
+    @PutMapping("/movie-sessions")
+    public void addMovieSession(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+        User user = userService.get(userId);
+        MovieSession movieSession = movieSessionService.get(movieSessionId);
+        shoppingCartService.addSession(movieSession, user);
+    }
```

Comment:
you must return here ShoppingCartResponseDto. And you can wite this code in one line)